cmd/snapd/main.go

```diff
@@ -49,6 +49,12 @@ func init() {
 func main() {
 	cmd.ExecInSnapdOrCoreSnap()
 	if err := run(); err != nil {
+		if err == daemon.ErrRestartSocket {
+			// Note that we don't prepend: "error: " here because
+			// ErrRestartSocket is not an error as such.
+			fmt.Fprintf(os.Stdout, "%v\n", err)
```

Comment:
Nitpick: perhaps we could add a comment that explains why the unexpected combination of fprintf + stdout is used (for testing)